addons/bestja_estimation_reports/models.py

```diff
@@ -144,10 +162,16 @@ class EstimationReport(models.Model):
         store=True,
     )
 
+    responsible_project = fields.Many2one(
+        'bestja.project',
+        string=u"projekt odpowiedzialny",
```

Comment:
There is another field added above with the same name, but different label. This is inconsistent.